SU-SWS/saa_victoria_subtheme
# Adapt 380  special font styles

# READY FOR REVIEW

# Summary
- set linked header styles in wysiwyg, and plain links in wysiwyg, somehow those are different than cta links..

# Review By (Date)
- When does this need to be reviewed by?

# Urgency
- minimal

# Steps to Test

1. pull down branch
1. check that it looks like calluna and proxima nova fonts are being applied on the page (calluna on headings and drop-cap big letter, splash text and callout text
2. proxima nova should be used for all regular text besides those cases ^

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
